src/app/cars/entities/car.ts

```diff
@@ -0,0 +1,6 @@
+export class Car {
+    id: number;
```

Comment:
id should be optional, not always Car model will contain the id and will still be consider as a Car (id?: number;)

Reply:
Changed id to optional